packages/maskbook/src/plugins/BarnBridge/Import/BarnBridgeFrontEnd/PortfolioBalance.tsx

```diff
@@ -0,0 +1,96 @@
+import { makeStyles } from '@masknet/theme'
+
+import { Typography, Divider, LinearProgress } from '@material-ui/core'
+import { formatUSDValue } from './web3/utils'
+import { useI18N } from '../../../../utils/i18n-next-ui'
+import { COLOR_SY_SENIOR_TEXT, COLOR_SY_JUNIOR_TEXT } from '../../constants'
+
+const s = makeStyles()((theme) => ({
+    progress: {
+        display: 'flex',
+        marginTop: 24,
+        '& .MuiLinearProgress-colorPrimary': {
+            backgroundColor: COLOR_SY_JUNIOR_TEXT,
+        },
+        '& .MuiLinearProgress-barColorPrimary': {
+            backgroundColor: COLOR_SY_SENIOR_TEXT,
+        },
+    },
+    dataColumn: {
+        paddingLeft: 16,
+        position: 'relative',
+        backgroundColor: 'var(--color)',
+        borderRadius: '50%',
+        height: 8,
+        left: 0,
+        top: 4,
+        width: 8,
+    },
+    portfolioAmountContainer: {
+        display: 'flex',
+        padding: 10,
+        paddingLeft: 30,
+        paddingRight: 30,
+        justifyContent: 'space-between',
+        alignSelf: 'space-between',
+        alignContent: 'space-between',
+        alignItems: 'space-between',
+        flex: 1,
+        flexDirection: 'row',
+    },
+}))
+
+type Props = {
+    total?: number
+    data: [[string, number | undefined], [string, number | undefined]]
```

Comment:
plz use labeled tuple elements feature, update the type definition

see https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements

Reply:
```ts
type SubItem = [label: string, value: number | undefined]
```

```suggestion
    data: [SubItem, SubItem]
```